frontend/src/components/user_page/UserPageLayout.tsx

```diff
@@ -22,17 +27,26 @@ const UserPageLayout: FunctionComponent = () => {
   const [loading, setLoading] = useState(true);
   const [notFound, setNotFound] = useState(false);
   const [user, setUser] = useState<null | User>(null);
-  //const [techs, setTechs] = useState();
+  const [selectedTech, setSelectedTech] = useState<Tech | null>();
+  const [search, setSearch] = useState("");
+  const [projects, setProjects] = useState<Project[] | null>();
+
+  const sessionUser = useSession().user;
 
   useEffect(() => {
     const getUser = async (username: string) => {
       const resp = await fetch("/api/v1/search/" + username);
       if (resp.status === 200) {
         const user = await resp.json();
         setUser(user);
-        //const arr = [a]
-        //if(user.techs)Object.keys(user.techs).forEach(key => arr.push({name: key, value: user.techs[key]}))
-        //setTechs(arr);
+        if (user.techs && user.techs.length > 0) {
+          setSelectedTech(user.techs[0]);
+          const resp = await fetch(
+            `/api/v1/projects/${user.id}/tech/${user.techs[0]?.id}`
```

Comment:
If we change the backend endpoint, we'll also want to change this to something like:
```
fetch(`/api/v1/users/${user.id}/projects?with_tech_id=${user.techs[0]?.id}`)
```